magenta/models/shared/melody_rnn_generate.py

```diff
@@ -60,6 +60,11 @@
                           'the unaltered softmax probabilities, greater than '
                           '1.0 makes melodies more random, less than 1.0 makes '
                           'melodies less random.')
+tf.app.flags.DEFINE_integer('bpm', -1,
```

Comment:
I think the idea is that this flag will override the bpm in a primer midi file if given.